src/PackageVersions/VersionsCommand.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace PackageVersions;
+
+use Composer\Command\BaseCommand;
+use Symfony\Component\Console\Input\InputArgument;
```

Comment:
This requires adding `symfony/console` to the package. That's not really in my plans at the moment.

Yes, I know it's bundled in composer, but no, I won't use it without an explicit `composer.json` entry for it.